sdk/python/src/dagger/mod/_module.py

```diff
@@ -197,6 +197,7 @@ def get_resolver(
     def __call__(self) -> None:
         if self._log_level is not None:
             configure_logging(self._log_level)
+        telemetry.initialize()
```

Comment:
Just ensuring that telemetry is initialized early instead of when using a custom span. There used to be custom root spans in this file, which took care of this, but those were removed in https://github.com/dagger/dagger/pull/7315.